llvm/lib/Target/RISCV/RISCVInstrInfo.td

```diff
@@ -524,7 +524,7 @@ class HLoad_r<bits<7> funct7, bits<5> funct5, string opcodestr>
 // Operands for stores are in the order srcreg, base, offset rather than
 // reflecting the order these fields are specified in the instruction
 // encoding.
-let hasSideEffects = 0, mayLoad = 0, mayStore = 1 in {
+let hasSideEffects = 0, mayLoad = 0, mayStore = 1, UseNamedOperandTable = 1 in {
```

Comment:
What is this needed for?

Reply:
'UseNamedOperandTable = 1' builds a table that's used to find instruction operand by its name in td

Reply:
I couldn't find any use of the table in your patch. Can you point me to it?